### PR TITLE
Docs Update: Add Reown as one of the Dev Tools available on Rootstock

### PR DESCRIPTION
**Pull request by rohit-710**
## Title
Adding Reown as one of the Dev Tools available on Rootstock for Account Abstraction and User Onboarding.

## Description
Reown provides fully open sourced and free to use SDKs for both Web3 apps and Web3 wallets. Reown also supports Rootstock, which means developers building a Web3 app on Rootstock can simply use Reown to power end-to-end wallet interactions on their app. 

## Testing

Tested locally.

## Checklist

- [x] I have read and understood the contributing guidelines.
- [x] I have followed the style guide and formatting guidelines.
- [x] I have added appropriate comments to explain the changes.
- [x] I have tested my changes thoroughly.

### CODE REVIEW

File: docs/05-dev-tools/account-abstraction/reown.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 102
+sidebar_label: Reown
+title: Account Abstraction using Reown
+description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
+---
+
+# Reown (prev. known as WalletConnect)
```
Comment (owans) [inline docs/05-dev-tools/account-abstraction/reown.md]:
```suggestion
```

File: docs/05-dev-tools/account-abstraction/reown.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
+
+### AppKit
```
Comment (owans) [inline docs/05-dev-tools/account-abstraction/reown.md]:
```suggestion
## AppKit
```

File: docs/05-dev-tools/user-onboarding/reown.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 101
+sidebar_label: Reown
+title: User Onboarding with Reown
+description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
+---
+
+# Reown (prev. known as WalletConnect)
```
Comment (owans) [inline docs/05-dev-tools/user-onboarding/reown.md]:
```suggestion
```

File: docs/05-dev-tools/user-onboarding/reown.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
```
Comment (owans) [inline docs/05-dev-tools/user-onboarding/reown.md]:
```suggestion
**[Reown](https://reown.com/?utm_source=rootstock&utm_medium=docs&utm_campaign=backlinks)** provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure.
```

File: docs/05-dev-tools/user-onboarding/reown.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Reown has two major product offerings, they are, **AppKit** and **WalletKit**.
+
+### AppKit
```
Comment (owans) [inline docs/05-dev-tools/user-onboarding/reown.md]:
```suggestion
## AppKit
```

File: docs/05-dev-tools/user-onboarding/reown.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+AppKit is a powerful, free, and fully open-source SDK for developers looking to integrate wallet connections and other Web3 functionalities into their apps on any EVM and non-EVM chain. In just a few simple steps, you can provide your users with seamless wallet access, one-click authentication, social logins, and notifications—streamlining their experience while enabling advanced features like on-ramp functionality, in-app token swaps and smart accounts.
+
+### WalletKit
```
Comment (owans) [inline docs/05-dev-tools/user-onboarding/reown.md]:
```suggestion
## WalletKit
```

File: docs/05-dev-tools/user-onboarding/index.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+Web3 onboarding often faces challenges due to complex wallet setups, unfamiliarity with blockchain concepts, and intimidating security practices. To drive mainstream adoption, it’s crucial to offer intuitive solutions that reduce friction while maintaining security. Providing well thought out end-to-end wallet interaction flows is a key approach to make Web3 more accessible for everyday users.
+
+## User Onboarding Solutions on Rootstock 
+
+<CardsGrid>
+  <CardsGridItem
+    title="Reown"
+    subtitle="User Onboarding"
+    color="green"
+    description="Reown (prev. known as WalletConnect) gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```
Comment (owans) [inline docs/05-dev-tools/user-onboarding/index.md]:
```suggestion
    description="Reown (prev. known as WalletConnect) provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```

File: docs/05-dev-tools/user-onboarding/reown.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 101
+sidebar_label: Reown
+title: User Onboarding with Reown
+description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
```
Comment (owans) [inline docs/05-dev-tools/user-onboarding/reown.md]:
```suggestion
tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, wallet connections, wallet infrastructure]
```

File: docs/05-dev-tools/account-abstraction/reown.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 102
+sidebar_label: Reown
+title: Account Abstraction using Reown
+description: "Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure." 
+tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, solana, wallet connections, wallet infrastructure]
```
Comment (owans) [inline docs/05-dev-tools/account-abstraction/reown.md]:
```suggestion
tags: [reown, account abstraction, developer tools, rootstock, ethereum, dApps, wallets, wallet connections, wallet infrastructure]
```

**Comment by owans** [comment]
Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.

**Comment by rohit-710** [comment]
> Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.

Hey @owans, thank you review the PR. I just addressed all of your comments. Let me know if it looks good to go now 🙏

File: docs/05-dev-tools/account-abstraction/index.md [hunk 10]
```diff
@@ -59,14 +59,14 @@ The Rootstock network is innovating in the account abstraction space, here are s
...
-    title="Gelato"
+  <CardsGridItem
+    title="Reown"
     subtitle="account abstraction"
     color="green"
-    description="Deploy production-grade & fully-serviced L2 rollups on Rootstock, natively integrated with tools like oracles, bridges, data indexers and Account Abstraction. It is supported on Mainnet."
-    linkHref="/dev-tools/dev-environments/gelato/"
-    linkTitle="Build with Gelato"
-  /> -->
+    description="Reown (prev. known as WalletConnect) gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```
Comment (owans) [inline docs/05-dev-tools/account-abstraction/index.md]:
```suggestion
    description="Reown (prev. known as WalletConnect) gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```
```suggestion
    description="Reown (prev. known as WalletConnect) provides developers with the tools to build user experiences that make digital ownership effortless, intuitive, and secure."
```

**Comment by owans** [comment]
> > Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.
> 
> Hey @owans, thank you review the PR. I just addressed all of your comments. Let me know if it looks good to go now 🙏

Thanks @rohit-710 Just one minor comment remaining.

This contribution also qualifies for the ongoing Rootstock Hacktivator Program: https://dev.rootstock.io/resources/contribute/hacktivator-program/

You can submit this PR here: https://forms.gle/aF9pFMfRyoygfzJWA

Once again. Thank you for contributing!

**Comment by rohit-710** [comment]
> > > Hi @rohit-710 Thank you for contributing to the docs and adding Reown DevTool. Please attend to the comments and I will merge.
> > 
> > 
> > Hey @owans, thank you review the PR. I just addressed all of your comments. Let me know if it looks good to go now 🙏
> 
> Thanks @rohit-710 Just one minor comment remaining.
> 
> This contribution also qualifies for the ongoing Rootstock Hacktivator Program: https://dev.rootstock.io/resources/contribute/hacktivator-program/
> 
> You can submit this PR here: https://forms.gle/aF9pFMfRyoygfzJWA
> 
> Once again. Thank you for contributing!

Awesome, thanks for letting me know! I missed that comment, just made a commit for the same.

**Review by owans — APPROVED** [review]
LGTM!

**Pull request merged**